NewForce-at-Mountwest/bangazon-site-the-blue-ferrets
# View Order

# Description
Created a page for the current shopping cart, its contents, and their prices. Adjusted the info displayed on the Products Index and Details views. Removed the "create new" button on the Index page, as this is now accessible via the "Sell Something" link.

## Related Tickets
https://trello.com/c/wsQiFcwb/22-missing-15-user-can-view-an-order

## Testing Instructions

- Git add and git commit current branch
- Git checkout master
- Git fetch --all
- Git checkout sarah1
- Launch browser with debugger or ctrl F5
- Make sure you are logged out
- Nav bar should only have 'Home' and 'Privacy' as options
- Log in as admin
- Nav bar should now have 'Product Types', 'Sell Something', and 'View Cart'
- Click view cart
- Should see 2 products, their price, and total
